packages/patternfly-4/react-core/src/components/Wizard/Wizard.tsx

```diff
@@ -91,57 +91,54 @@ export interface WizardProps {
   ariaLabelCloseButton?: string;
 }
 
-export class Wizard extends React.Component<WizardProps> {
-  private static currentId = 0;
-  private static defaultProps = {
+interface WizardState {
+  currentStep: number;
+  isNavOpen: boolean;
+}
+
+export class Wizard extends React.Component<WizardProps, WizardState> {
+  static currentId = 0;
+  static defaultProps = {
     isOpen: false,
     isCompactNav: false,
     isFullHeight: false,
     isFullWidth: false,
-    width: null,
```

Comment:
Are these removed because they are assumed to be null if you don't define them?

Reply:
Correct.